AsyncSwift/Views/SessionView.swift

```diff
@@ -0,0 +1,77 @@
+//
+//  SessionView.swift
+//  AsyncSwift
+//
+//  Created by Kim Insub on 2022/09/08.
+//
+
+import SwiftUI
+
+struct SessionView: View {
+
+    private let session: EventModel.Session
+
+    init(session: EventModel.Session) {
+        self.session = session
+    }
+
+    var body: some View {
+        ScrollView {
+            CustomDivider()
+                .padding(.top, 10)
+                .padding(.bottom, 4)
+            sessionDetail
+            speakerDetail
+        }
+        .navigationTitle("Session")
+    }
+}
+
+private extension SessionView {
+
+    var sessionDetail: some View {
+        VStack(alignment: .leading) {
+            Text(session.title)
+                .font(.system(size: 20))
+                .fontWeight(.semibold)
+                .padding(.vertical, 24)
+            VStack(alignment: .leading, spacing: 8) {
+                ForEach(session.description, id: \.self) { paragraph in
+                    Text(paragraph.content)
+                }
+            }
+            .padding(.bottom, 80)
+        }
+        .padding(.horizontal, 24)
+    }
+
+    var speakerDetail: some View {
+        VStack(alignment: .leading, spacing: 4) {
+            HStack {
+                Circle()
+                    .frame(width: 80, height: 80, alignment: .leading)
+                    .padding(.vertical, 24)
+                Spacer()
+            }
```

Comment:
상단 VStack에서의 allignment가 leading으로 설정되어있으니 필요 없지 않을까요?

```suggestion

                Circle()
                    .frame(width: 80, height: 80, alignment: .leading)
                    .padding(.vertical, 24)

```

Reply:
상속되지 않습니다
고로 다시 한번 할당해주어야합니다

Reply:
해보시면 될겁니다!(해봤습니다) + 서클에 alignment도 따로 설정이 필요하지 않습니다.